### ISSUE TITLE
Accessibility: The label 'Example' is not associated with the input element

### ISSUE THREAD

**Issue by saskiaw** [Redocly/redoc#2520]
**Describe the problem to be solved**
"The label 'Example' is not associated with the input element. In the code, the label should be linked to the input fields Museum opening hours & The museum is closed ."

![image](https://github.com/Redocly/redoc/assets/74241382/6b3213a6-8fe9-42c3-9fc2-02b37681ae4f)
